bughouse_console/src/secret_database.rs

```diff
@@ -170,6 +226,32 @@ where
         Ok(())
     }
 
+    async fn set_logged_in_session(
+        &self, id: &SessionId, user_name: Option<String>, expiration: OffsetDateTime,
+    ) -> anyhow::Result<()> {
+        sqlx::query::<DB>(
+            "INSERT INTO sessions(session_id, user_name, expires_at)
+            VALUES($1, $2, $3)
+            ON CONFLICT(session_id)
```

Comment:
Could `INSERT OR REPLACE` be used instead?

Reply:
Sure, I don't have a preference, and in this case I don't understand which one would be better. My intuition was, upsert will be faster with no downsides.

Another consideration was, if we ever want this table partially updated, upsert will be the way to go. Changed to `INSERT OR REPLACE`.